### DISCUSSION TITLE
cannot login due to not configure 2FA

### DISCUSSION THREAD

**Discussion by YukeyWYK** [community/community#142445] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a company github account. I can't log in now because I didn't set 2FA in time. What should I do?

**Comment by fxrdhan** [comment]
Hi there! If you're unable to log in due to not setting up 2FA before the deadline, here are the steps you can take:

1. Contact your organization's GitHub admin/owner - they can:
   - Temporarily disable the 2FA requirement for your account
   - Help reset your access so you can set up 2FA
   - Provide guidance on your company's specific 2FA policies

2. If you don't know who your GitHub admin is:
   - Check with your IT department or team lead
   - Look up your organization's settings page (if you can access it)
   - Contact your company's technical support

3. Once you regain access, immediately set up 2FA using either:
   - Authentication app (recommended) like Google Authenticator, Microsoft Authenticator, or Authy
   - SMS verification (as backup)
   - Security keys (for enhanced security)

As a reminder, GitHub now requires 2FA for all contributing developers as part of their security policy. This is mandatory and cannot be permanently bypassed.

**Comment by thomassiard** [comment]
If you’re unable to log into your company GitHub account due to missing the two-factor authentication (2FA) setup, here are steps you can take:

Recovery Options:

First, check if you have recovery codes that were generated when you set up the account. These codes can help you regain access.
Contact GitHub Support:

If you don’t have recovery codes or backup methods, contact [GitHub Support](https://support.github.com/contact) and explain the situation. GitHub Support can assist with 2FA recovery under specific conditions. You’ll need to verify your identity as the account owner, so be prepared to provide any details or verification requested.
Organization Owners:

If other organization owners are set up on the account, they may be able to adjust account access or assist with account recovery.
Following these steps should help you regain access. Good luck!
